phpcs.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<ruleset name="SimpleSAMLphp SAML2 ruleset">
+    <config name="ignore_warnings_on_exit" value="1"/>
+
```

Comment:
do we need this file?

Reply:
This file is included in the core SSP modules, so I add this file to RCIAM modules in order to keep up with the code style of SimpleSAMLphp